package/PACKAGING.md

```diff
@@ -8,10 +8,6 @@ If you'd like to build an installable bundle of VocabHunter for Mac, Linux or Wi
 
 You will need to install JDK 11 to build the installable bundle of VocabHunter.  The version provided by [AdoptOpenJDK](https://adoptopenjdk.net/) is an excellent choice for this.
 
-### OpenJFX 11
-
-Get the JavaFX 11 JMOD files from [the Gluon download page](https://gluonhq.com/products/javafx/).  Choose the right option for your platform.  For example, for Linux pick the file listed as "JavaFX Linux jmods".  Unzip the file and save the resulting directory somewhere on your system.
-
 ### Java Packager
 
 Download the Java Packager.  Choose the right link for your platform from those listed [on this page](https://mail.openjdk.java.net/pipermail/openjfx-dev/2018-September/022500.html).  For Mac and Linux, unpack the zip file store the directory somewhere on your system.  For Windows, copy the two files into the following directories:
```

Comment:
It's okay to stick to this version as discussed, but maybe you can say:

> While the official `jpackage` tool it is being developed as part of [JEP 343](http://jdk.java.net/jpackage/) for Java 13, a very early access of it for JDK 11 can be downloaded from ...

Reply:
Good idea. I've slightly reworded it to keep the flow of the document as a list of instructions and also replaced the HTTP URL with an HTTPS equivalent.  Take a look and see what you think.

Reply:
Fine by me, +1

Reply:
Good! 

The PR is looking good I think. It works on Mac and Linux.  I'm going to test on a Windows VM to make sure all is well there too.